### PR TITLE
Update PR: Uninstall CTT PowerShell Profile

### PR DESCRIPTION
**Pull request by Cryostrixx**
## Contributor Note
This is a work-in-progress PR. It will be updated as I acquire feedback in PR/**Uninstall CTT PowerShell Profile**.
Feedback on this PR is not a requirement until I manually update the pull request's status to **ready for review**.

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Documentation update
- [x] Refactoring
- [ ] Hotfix
- [ ] Security patch
- [ ] UI/UX improvement

## Description
Work-in-progress pull request for making any requested changes to the files that are included within PR #3012.

## Testing
N/A

## Impact
N/A

## Issue related to PR
N/A

## Additional Information
N/A

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

File: functions/public/Invoke-WPFButton.ps1 [hunk 1]
```diff
@@ -50,13 +50,15 @@ function Invoke-WPFButton {
         "WPFFixesNetwork" {Invoke-WPFFixesNetwork}
         "WPFUpdatesdisable" {Invoke-WPFUpdatesdisable}
         "WPFUpdatessecurity" {Invoke-WPFUpdatessecurity}
+        "WPFWinUtilShortcut" {Invoke-WPFShortcut -ShortcutToAdd "WinUtil" -RunAsAdmin $true}
```
Comment (og-mrk) [inline functions/public/Invoke-WPFButton.ps1:53]: This feature (Creating a Shortcut) have been removed in commit c186642998a77bcb95e59c46dd272f100ab50423, [link to exact line](https://github.com/ChrisTitusTech/winutil/commit/c186642998a77bcb95e59c46dd272f100ab50423#diff-70bdd2b079dc5748db02b2676484003945a7d561d8d903b3c742dadd69d30b24L53).

Comment (Cryostrixx) [reply functions/public/Invoke-WPFButton.ps1:53]: I should've done that anyway... I'm not sure how I missed any of this. Would it be best to make a new PR or should I just push to this one?

Comment (og-mrk) [reply functions/public/Invoke-WPFButton.ps1]: I recommend pushing to this branch and keeping this PR open, as it's currently a draft/work-in-progress PR (as stated by you). Also, things like this happen from time to time, so no worries 😄

Comment (Cryostrixx) [reply functions/public/Invoke-WPFButton.ps1]: I know... But I still think a new PR is the best option, plus I can't wrap my head around conflict resolution.

**Pull request closed without merging**